src/test/resources/db/migration/V1__init_db.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE public.new_user (
+	id text NOT NULL,
+	first_name text NULL,
+	last_name text NULL,
```

Comment:
Keycloak calls these field in plural like: 'first_names' and 'last_names', how do you think we are going to handle multiple first or last names?

Reply:
There can be a space in the text field so "Tom Jan Dijkema Van Dijk" Would split into: "Tom Jan" and "Dijkema Van Dijk". Don't really see an issue there, same as Elvis is doing it.